gradle-baseline-java-config/resources/eclipse/org.eclipse.jdt.core.prefs

```diff
@@ -93,8 +92,8 @@ org.eclipse.jdt.core.formatter.comment.preserve_white_space_between_code_and_lin
 org.eclipse.jdt.core.formatter.compact_else_if=true
 org.eclipse.jdt.core.formatter.continuation_indentation=2
 org.eclipse.jdt.core.formatter.continuation_indentation_for_array_initializer=2
-org.eclipse.jdt.core.formatter.disabling_tag=@formatter\\:off
-org.eclipse.jdt.core.formatter.enabling_tag=@formatter\\:on
+# org.eclipse.jdt.core.formatter.disabling_tag=@formatter\\:off
+# org.eclipse.jdt.core.formatter.enabling_tag=@formatter\\:on
```

Comment:
Do these changes have a different effect than deleting the lines? If not, let's delete them.